test/integration/confirmations/transactions/alerts.test.tsx

```diff
@@ -411,4 +411,73 @@ describe('Contract Interaction Confirmation Alerts', () => {
       await screen.findByTestId('alert-modal-action-showGasFeeModal'),
     ).toHaveTextContent('Update gas options');
   });
+
+  it('displays the alert for signing and submitting alerts', async () => {
```

Comment:
The difference between the lines changed is because this test only exists in the `main` branch.